docs/user_guide/datasets.rst

```diff
@@ -0,0 +1,187 @@
+
+    >>> print(f"Demographic parity difference for:\nB: {DP_B}\nLSTAT: {DP_LSTAT}")
+    Demographic parity difference for:
+    B: 0.0901639344262295
+    LSTAT: 0.8877297565822156
+
+Checking the demographic parity differences shows that neither variable has a demographic parity at zero, implying a different selection rate across groups. The next series of tables dives deeper into the breakdown of various metrics by group. The proportion of Blacks higher than the median is associated with a higher false positve rate. *B* == True is also associated with a slightly lower precision. The accuracy, recall, and selection rate when *LSTAT* == True all are lower than when *LSTAT* is False. These results indicate that our simple model is worse at predicting an outcome for individuals in our constructed "lower status" category.
+
+    >>> metrics = {
+    ... 'accuracy': skm.accuracy_score,
+    ... 'precision': skm.precision_score,
+    ... 'recall': skm.recall_score,
+    ... 'false positive rate': false_positive_rate,
+    ... 'true positive rate': true_positive_rate,
+    ... 'selection rate': selection_rate, 
+    ... 'count': count}
+    >>> grouped_metric = MetricFrame(metrics=metrics,
+    ... y_true=y_test, 
+    ... y_pred=predicted,
+    ... sensitive_features=X_test["B"])
+    >>> print(grouped_metric.by_group)
+        accuracy precision    recall false positive rate true positive rate  \
+    B                                                                            
+    False  0.852459      0.92  0.766667            0.064516           0.766667   
+    True   0.863636  0.909091  0.833333                 0.1           0.833333   
+
+        selection rate count  
+    B                           
+    False       0.409836    61  
+    True             0.5    66  
+
+    >>> grouped_metric = MetricFrame(metrics=metrics,
+    ... y_true=y_test,
+    ... y_pred=predicted,
+    ... sensitive_features=X_test["LSTAT"])
+    >>> print(grouped_metric.by_group)
+        accuracy precision    recall false positive rate true positive rate  \
+    LSTAT                                                                        
+    False  0.901639  0.910714  0.980769            0.555556           0.980769   
+    True   0.818182       1.0  0.142857                 0.0           0.142857   
+
+        selection rate count  
+    LSTAT                       
+    False       0.918033    61  
+    True        0.030303    66  
+
+
+.. _discussion:
+
+Discussion
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+The Boston housing dataset is rife with ethical issues, and in general, we strongly discourage using it. We've kept it in fairlearn because of its potential as a teaching tool for how to deal with ethical issues in a dataset. There are ways to `debias the data <https://scikit-lego.netlify.app/fairness.html>`, but other benchmark datasets exist that do not present these issues.
```

Comment:
I would prefer to not use the word 'debias' here.

Reply:
Some more context on this: https://fairlearn.org/main/contributor_guide/how_to_talk_about_fairness.html

What they probably mean by debias is to try to mitigate some of the issues. Since it's not possible to fully "debias" we try to stay away from that term as it may lead people to believe that it is indeed possible.

Reply:
I replaced "debias" with "remove correlations between sensitive features and the remaining columns" in a recent commit. Please let me know if that edit is satisfactory or if you have other suggestions.